test/e2e/shared/common.go

```diff
@@ -207,13 +207,13 @@ func dumpSpecResources(ctx context.Context, e2eCtx *E2EContext, namespace *corev
 	})
 }
 
-func Byf(format string, a ...interface{}) {
-	By("[" + time.Now().Format(time.RFC3339) + "] " + fmt.Sprintf(format, a...))
+func Logf(format string, a ...interface{}) {
+	fmt.Fprintf(GinkgoWriter, "["+time.Now().Format(time.RFC3339)+"] "+format+"\n", a...)
```

Comment:
this is the key thing we refer from CAPZ? 
isntead of By, we use `fmt.Fprintf(GinkgoWriter`, correct?

Reply:
Correct!